pkg/cluster-runtime/apps.go

```diff
@@ -90,6 +94,11 @@ func (i AppInstaller) Launch(master0 net.IP, launchCmds []string) error {
 		ex         = shell.NewLex('\\')
 	)
 
+	imagePolicyEngine, err := kyverno.NewKyvernoImagePolicyEngine()
```

Comment:
make image policy engine effect only when the image has labels `app.alpha.sealer.io/image-policy-plugin`= "kyverno"